src/css/override_vars.scss

```diff
@@ -14,34 +15,101 @@ $color-primary-alt-darkest:  darken($color-primary-alt, 70%);
 $color-primary-alt-light:    #77b6ff;
 $color-primary-alt-lightest: #d9f1ff;
 
-$color-secondary:            #e74c3c;
+$color-red:            #cb4f03;
+$color-secondary:            $color-red;
 $color-secondary-dark:       #b0392e;
 $color-secondary-darkest:    darken($color-secondary, 75%);
 $color-secondary-light:      #f3736e;
 $color-secondary-lightest:   #deeeef; // DEEF!
 
-$color-focus:                rgba(255, 221, 0, 0.55);
-$color-visited:              #4c2c92;
+$color-warning:              #cb4f03;
+$color-error:                #cb4f03;
 
-$color-gold:                 #f8ba1c;
+$color-visited:              $color-blue;
 
-$color-finish:               #0daf02;
-$color-warning:              #cb4f03;
-$color-error:                #f16031;
+$color-gold:                 #f4b400;
+$color-focus:                $color-gold;
+
+$color-green:               #0daf00;
+$color-finish:               $color-green;
 
-$color-gray:                 #595959;
-$color-gray-dark:            darken($color-gray, 20%);
+$color-darkgray: #555;
+$color-mediumgray: #8e8e8e;
+$color-lightgray: #d3d3d3;
+$color-textblack: #333;
+$color-trueblack: #000;
+
+$color-gray-dark:       $color-darkgray; // redundant to prevent breaking
+$color-gray:             $color-mediumgray;
 
 $color-disclaimer-text:      #000000;
 $color-disclaimer-link:      #0071bc;
 
-$color-base: $color-gray;
-$text-max-width:    660px;
+$color-base:                 $color-textblack;
+
+$color-cream:     #f5f5f3;
+$color-background-light:     $color-cream;
+
+// typefaces
+$font-sans:                  '18Franklin', system, -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
+$font-serif:                 'Georgia', 'Cambria', 'Times New Roman', 'Times', serif;
+$font-mono:                  'Roboto Mono', 'Bitstream Vera Sans Mono', 'Consolas', 'Courier', monospace;
+
+// grid
+$grid: 8; // unitless for some calculations
```

Comment:
this value stands for the base whitespace scale multiplier (i.e. 8px) — it may not have any effect on the columnar "grid"